scheds/rust/scx_layered/src/main.rs

```diff
@@ -1584,6 +1584,9 @@ impl<'a> Scheduler<'a> {
         skel.struct_ops.layered_mut().exit_dump_len = opts.exit_dump_len;
 
         skel.maps.rodata_data.debug = opts.verbose as u32;
+        // Running scx_layered inside a PID namespace would break the
```

Comment:
I think if we write the namespace id as well we can translate it on the bpf side, we already have the [helpers](https://github.com/sched-ext/scx/blob/main/scheds/include/scx/namespace.bpf.h).

Reply:
Ah, cool, I'll do that later.